packages/braintree-utils/src/braintree.ts

```diff
@@ -1048,10 +1049,20 @@ export interface BraintreeFastlaneCardComponentField {
 }
 
 export interface BraintreeFastlaneTokenizeResult {
-    nonce: string;
-    details: BraintreeFastlaneTokenizeDetails;
-    description: string;
-    type: string;
+    id: string;
+    paymentSource: PaymentSource;
+}
+
+export interface PaymentSource {
+    card: CardPaymentSource;
+}
+
+export interface CardPaymentSource {
+    brand: string;
+    expiry: string; // "YYYY-MM"
+    lastDigits: string; // "1111"
+    name: string;
+    billingAddress: BillingAddress;
```

Comment:
It is not necessary to add this interfaces because you already have them in the file. Please take a look on `BraintreeFastlaneVaultedInstrument` interface.

Reply:
fixed